lib/index.d.ts

```diff
@@ -0,0 +1,644 @@
+/// <reference types="@panva/jose" />
```

Comment:
someone care to explain to a newbie here why this needs to be referenced but `got` doesn't?

Reply:
It should be included. I'll update.

https://stackoverflow.com/questions/48216163/what-does-reference-types-node-mean